packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+    <p :class="[$style['c-restaurantCard-rating']]">
+        <!-- Icons -->
+        <star-empty-icon
+            v-if="noRatingsAvailable"
+            aria-hidden="true"
+            data-test-id="ratings-star-empty"
+            :class="[$style['c-restaurantCard-ratingStar']]" />
+
+        <star-filled-icon
+            v-else
+            aria-hidden="true"
+            data-test-id="ratings-star-filled"
+            :class="[$style['c-restaurantCard-ratingStar']]" />
+
+        <!-- No ratings message -->
+        <span
+            v-if="noRatingsAvailable"
+            data-test-id="no-ratings-message"
+            :class="[$style['c-restaurantCard-ratingNotRatedMsg']]">
+            {{ notRatedMessage }}
+        </span>
+
+        <template v-else>
+            <!-- Screenreader message (hidden) -->
+            <!-- NOTE: The accessibility requires for both this and the restaurant card as a whole
+                 are still being discovered. Therefore this is subject to change -->
+            <span
+                data-test-id="ratings-summary-message"
+                class="is-visuallyHidden">
+                {{ accessibleMessage }}
+            </span>
+
+            <!-- Mean value -->
+            <data
+                :class="[$style['c-restaurantCard-ratingMean']]"
+                :value="mean"
+                data-test-id="ratings-mean-value"
+                aria-hidden="true">
+                {{ meanFormatted }}
+            </data>
+
+            <span
+                aria-hidden="true"
+                :class="[$style['c-restaurantCard-ratingDivider']]">&#47;</span>
+
+            <!-- Max value -->
+            <data
+                :class="[$style['c-restaurantCard-ratingOutOf']]"
+                :value="ratingsMax"
+                aria-hidden="true">
+                {{ ratingsMax }}
+            </data>
+
+            <!-- Number of ratings/Own rating message -->
+            <span aria-hidden="true">
+                &#40;
+                <data
+                    v-if="!isOwnRating"
+                    data-test-id="rating-count"
+                    :class="[$style['c-restaurantCard-ratingCount']]"
+                    :value="count">
+                    {{ count }}
+                </data>
+                <span
+                    v-else
+                    data-test-id="rating-own-rating-message"
+                    :class="[$style['c-restaurantCard-ratingCount']]">
+                    {{ isOwnRatingMessage }}
+                </span>
+                &#41;
+            </span>
+        </template>
+    </p>
+</template>
+
+<script>
+import { StarEmptyIcon, StarFilledIcon } from '@justeat/f-vue-icons';
+
+const RATINGS_MAX = 6;
+
+export default {
+    name: 'RestaurantRating',
+    components: {
+        StarEmptyIcon,
+        StarFilledIcon
+    },
+    props: {
+        isOwnRating: {
```

Comment:
Would be good to add some comments around these props., so that if we do pull it out into its own component, those comments can be used to populate the README (plus it'll give better context for those new to the component here too).